packages/web-extension/src/walletManager/walletManager.types.ts

```diff
@@ -0,0 +1,34 @@
+import { AsyncKeyAgent } from '@cardano-sdk/key-management';
+import { ObservableWallet, storage } from '@cardano-sdk/wallet';
+
+export interface WalletManagerProps {
+  walletName: string;
+}
+
+export interface WalletManagerActivateProps {
+  observableWalletName: string;
+  cardanoServicesUrl?: string;
```

Comment:
It's a little easier to reason about if it's typed with non-nullable props
```suggestion
  __typename: ProviderType.CardanoServices, // could omit this for now since it's the only type
  cardanoServicesUrl: string; // this should be a required prop
```

Reply:
Consider suggestion [in this comment](https://github.com/input-output-hk/cardano-js-sdk/pull/502#discussion_r1013999100) instead

Reply:
Done here: [4d319c5](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/4d319c5241651b905ef6fb7b77000a855f60eb96)